client/utils.lua

```diff
@@ -64,10 +66,85 @@ if GetConvarInt('ox_target:drawSprite', 1) == 1 then
             end
         end
 
+
+
         return n > 0 and inRange, newZone
     end
 
+
+    function utils.getNearbyTarget(coords)
+        coords = coords or GetEntityCoords(PlayerPedId())
+        inRange = {}
+
+        local n = 0
+        local targetData  = getTargetData();
+        local activated   = getActivated();
+        local isActive    = state.isActive();
+
+        local getDistance = function(obj) 
+            return #(GetEntityCoords(PlayerPedId())-GetEntityCoords(obj))
+        end
+
+        local models = api.getModels();
+        for _, obj in pairs(GetGamePool("CObject")) do
+            local distance = getDistance(obj)
+            local model = GetEntityModel(obj);
+
+            if (DoesEntityExist(obj) and models[model] and (distance < 7)) then
```

Comment:
Oh, the object one checks if it's in models. Of course, vehicle and peds can also use models but don't; meanwhile this one never checks entities/localEntities.